satpy/etc/readers/amsr2_l2.yaml

```diff
@@ -0,0 +1,41 @@
+reader:
+  name: amsr2_l2
+  description: GCOM-W1 AMSR2 instrument HDF5 reader
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+  sensors: [amsr2]
+
+file_types:
+  amsr2_l2:
```

Comment:
If each product is in a separate file then we should either have a separate file type for each product.

Reply:
ok